asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -332,25 +405,40 @@ class PortfolioPlanner extends React.Component {
     }))
   }
 
-  createRegionName(regions) {
-    // returns a CSV string of names for the different regions
-    const names = regions.map(({ name }) => {
-      return name
-    }).join(',')
-
-    return names
-  }
-
   getDate() {
     const date = new Date(new Date().toString().split('GMT')[0]+' UTC').toISOString().split('T')[0]
     return date
   }
+  
+  changeSection(e) {
+    const newSection = e.target.value
+    this.setState(prevState => {
+      return {
+        ...prevState,
+        selectedSection: newSection
+      }
+    })
+  }
 
   render() {
-    // This filters on every re-render, so that this.state.remainingProjects can be the source of truth.
-    // Could also chain filtering here for other things (like "department" from the wireframe).
     const filteredRows = this.state.remainingProjects && this.state.remainingProjects.filter(project => {
       return project.description.toLowerCase().includes(this.state.filterText.toLowerCase())
+    }).filter(project => {
+      if (this.state.selectedSection !== '') {
```

Comment:
why not 

```
if (this.state.selectedSection) {
```

Reply:
done so here: https://github.com/fpdcc/ccfp-asset-dashboard/commit/aa9691fa5b92f138284e8198b24feaba889d84c8